### PR TITLE
new test 932160-8

### PR DESCRIPTION
**Pull request by dune73**
New test triggering only 932160 (for status page).

### CODE REVIEW

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml [hunk 1]
```diff
@@ -114,3 +114,19 @@ tests:
             protocol: "http"
           output:
             no_log_contains: "id \"932160\""
+  - test_title: 932160-8
+    desc: "Positive test: Triggering only this rule an no other one"
```
Comment (fzipi) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml:118]:
```suggestion
    desc: "Positive test: Triggering only this rule and no other one"
```

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml [hunk 2]
```diff
@@ -114,3 +114,19 @@ tests:
...
+    stages:
+      - stage:
+          input:
+            dest_addr: "127.0.0.1"
+            method: "POST"
```
Comment (fzipi) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml:123]:
```suggestion
            method: "POST"
            uri: "/post"
```

**Review by fzipi — CHANGES_REQUESTED** [review]

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml (hunk 1 above)
Comment (dune73) [reply tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml]: thanks

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml (hunk 2 above)
Comment (dune73) [reply tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932160.yaml:123]: thanks. fixed

**Review by fzipi — APPROVED** [review]

**Pull request merged**